string output_filename
---
sensor_msgs/RegionOfInterest face_position
